recipes/mosfit/meta.yml

```diff
@@ -0,0 +1,64 @@
+{% set name = "mosfit" %}
+{% set version = "0.4.7" %}
+{% set sha256 = "e63f1196524f0d59915fc8289cd8bd023999853e59ccf54254fc8783112ab901" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ mosfit }}-{{ 0.4.7 }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scipy
+    - astropy
+    - Cython
+    - emcee
+    - extinction
+    - inflect
+    - matplotlib
+    - mpi4py
+    - nbstripout
+    - numexpr
+    - palettable
+    - seaborn
+    - six
+  run:
+    - python
+
+test:
+  imports:
+    - mosfit
+
+about:
+  home: http://github.com/guillochon/mosfit
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Simple, fast, extensible JSON encoder/decoder for Python'
+
+  description: |
+    mosfit is a package that performs fitting of semi-analytical model predictions
+    to observed transient data. Data can be provided by the user, or can be pulled
```

Comment:
"transient data" is confusing if you don't know the astrophysical context; I suggest clarifying this.